tensorboard/plugins/interactive_inference/witwidget/notebook/jupyter/js/lib/wit.js

```diff
@@ -40,9 +41,14 @@ var WITView = widgets.DOMWidgetView.extend({
     const height = parseInt(
       this.model.attributes.layout.attributes.height, 10) - 20;
     const iframe = document.createElement('iframe');
-    const templateLocation =
-      window.__webpack_public_path__ + 'wit_jupyter.html';
-    const src = `<link rel="import" href="${templateLocation}">
+
+    // Adjust WIT html location if running in a jupyter notebook
+    // and not in jupyterlab.
+    if (document.querySelector('body').getAttribute('data-base-url') != null) {
+      witHtmlLocation = window.__webpack_public_path__ + 'wit_jupyter.html';
```

Comment:
I'd recommend renaming `__webpack_public_path__` to something like `__nbextension_path__`.

Reply:
done